hbase-it/pom.xml

```diff
@@ -106,6 +106,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
Unless the its run the diagnostics tools explicitly, it would be better to avoid this dependency.

Reply:
i was able to get rid of test-jar only as hbase-it extensively depends on diagnostics-tools
Refer below:
- IntegrationTestRegionReplicaPerf => Uses PerformanceEvaluation for test
- StripeCompactionsPerformanceEvaluation => Uses LoadTestDataGenerator 
- IntegrationTestIngestWithVisibilityLabels extends IntegrationTestIngest  => Uses LoadTestTool
- IntegrationTestIngest => Uses LoadTestTool
- IntegrationTestIngestWithACL extends IntegrationTestIngest => Uses LoadTestTool
- IntegrationTestTimeBoundedRequestsWithRegionReplicas => Uses LoadTestTool
- IntegrationTestLazyCfLoading extends LoadTestDataGenerator 
- IntegrationTestRegionReplicaReplication extends IntegrationTestIngest  
- IntegrationTestMTTR => Uses LoadTestTool
- IntegrationTestIngestWithMOB extends IntegrationTestIngest  => Uses LoadTestTool
- IntegrationTestTimeBoundedMultiGetRequestsWithRegionReplicas extends IntegrationTestIngest  => Uses LoadTestTool
- IntegrationTestIngestWithTags extends IntegrationTestIngest => Uses LoadTestTool